fdb-relational-core/src/main/java/com/apple/foundationdb/relational/recordlayer/query/QueryPlan.java

```diff
@@ -186,7 +187,7 @@ public PhysicalQueryPlan withQueryExecutionParameters(@Nonnull final QueryExecut
         public String explain() {
             final var executeProperties = queryExecutionParameters.getExecutionPropertiesBuilder();
             List<String> explainComponents = new ArrayList<>();
-            explainComponents.add(recordQueryPlan.toString());
+            explainComponents.add(ExplainPlanVisitor.toStringForExternalExplain(recordQueryPlan));
```

Comment:
We can improve this later on by incorporating different `EXPLAIN` verbosity levels in SQL, e.g. similar to PostgreSQL https://github.com/FoundationDB/fdb-record-layer/issues/3028